packages/beacon-node/src/node/nodejs.ts

```diff
@@ -213,6 +213,11 @@ export class BeaconNode {
     // Load persisted data from disk to in-memory caches
     await chain.loadFromDisk();
 
+    if (opts.chain.persistInvalidSszObjects && opts.chain.persistInvalidSszObjectsRetention !== 0) {
+      chain.prunePastInvalidSszObjects();
+      setInterval(() => chain.prunePastInvalidSszObjects(), 1000 * 60 * 60);
```

Comment:
This interval must be handled and clean gracefully on shutdown

Reply:
I used `signal.addEventListener("abort", [...])`, is that ok?